gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -275,12 +275,27 @@ private WriteObjectResponse doResumableUpload() throws IOException {
 
       boolean objectFinalized = false;
       while (!objectFinalized) {
+
+        if (responseObserver.isComplete()) {
+          // reset streams
+          responseObserver = new InsertChunkResponseObserver(uploadId, writeOffset);
+          call =
+              storageStub
+                  .getChannel()
+                  .newCall(StorageGrpc.getWriteObjectMethod(), stub.getCallOptions());
+          writeObjectRequestStreamObserver =
+              ClientCalls.asyncClientStreamingCall(call, responseObserver);
+          requestStreamObserver =
+              watchdog.watch(
+                  call,
+                  writeObjectRequestStreamObserver,
+                  Duration.ofSeconds(channelOptions.getGrpcWriteMessageTimeoutMillis()));
```

Comment:
Should it be `Duration.ofMillis`?

Reply:
Seems like it's been wrong from all along. Updated.